### ISSUE TITLE
[EDDSA] Highlight security/privacy trade-offs between RDF-CANON and JCS

### ISSUE THREAD

**Issue by msporny** [w3c/vc-data-integrity#195]
From the PING's review (https://github.com/w3cping/privacy-request/issues/120):
>The spec should highlight the security tradeoffs that occur between section 3.1 and section 3.2 or select one to avoid encountering issue 1 highlighted in the https://github.com/w3cping/privacy-request/issues/121#issuecomment-1638908803 . The security of section 3.1 relies upon https://www.w3.org/TR/rdf-canon/ which is light on the security and privacy sections as well. It appears it's undergoing a review at the moment as well with https://github.com/w3cping/privacy-request/issues/118

/cc @kdenhartog

**Comment by msporny** [comment]
This is a duplicate of #194. Marking pending close (as we'll address this in #194). If PR #199 is merged before then, this issue will be closed once that PR is merged.

**Comment by msporny** [comment]
PR #199 has been merged, closing.
